learning_journal_basic1/templates/entry.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends 'index.jinja2' %}
+{% block body %}
+
+  <form action="{{  request.route_url('entry_view') }}" method="POST">
+
+    New Title:
+    <br>
+    <input type="text" size="50" name="title"/><br>
```

Comment:
If this is meant to edit an existing entry, it is customary to show the current value for title and body.